cli/src/commands/payment_address.rs

```diff
@@ -95,6 +95,105 @@ pub mod list_command {
     }
 }
 
+pub mod sign_command {
+    use super::*;
+
+    command!(CommandMetadata::build("sign", "Create a proof of payment address control by signing an input and producing a signature.")
+                .add_required_param("address", "Payment address to use")
+                .add_required_param("input", "The input data to be signed")
+                .add_example("sign address=pav:sov:1a2b3c4d5e6f7g8h9ioplkjhgfdsfgbv input=910274529363984")
+                .finalize());
+
+    fn execute(ctx: &CommandContext, params: &CommandParams) -> Result<(), ()> {
+        trace!("execute >> ctx {:?} params {:?}", ctx, params);
+
+        let wallet_handle = ensure_opened_wallet_handle(&ctx)?;
+
+        let address = get_str_param("address", params).map_err(error_err!())?;
+        let input = get_str_param("input", params).map_err(error_err!())?;
```

Comment:
input in libindy API is raw bytes. Should we have an opportunity to sign any data in CLI, no only UTF-8 string?

Reply:
yes but this is not currently done in the CLI. You would have to either send it via a pipe or read it from a file or another read input method because the user couldn't pass raw bytes in its current state.

Reply:
I think we have a mechanism to read some value from a file for other commands.
btw I wouldn't like to grow this PR a lot. But if some technical debt is introduced - please create a follow-up ticket and refers to it from the TODO in codebase and parent ticket in Jira

Reply:
That's fine. I can create another ticket for that.